src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="Xamarin.Android.Tools.BootstrapTasks.GenerateProfile" AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" />
-  <Target Name="_Foo" BeforeTargets="CoreCompile">
+  <ItemGroup>
+    <SharedRuntimeBuildPath Include="..\..\bin\$(Configuration)\lib\xbuild-frameworks\MonoAndroid\" />
+    <_SharedRuntimeAssemblies Include="@(SharedRuntimeBuildPath->'%(Identity)\\v1.0\*.dll');@(SharedRuntimeBuildPath->'%(Identity)\\v6.0\*.dll')"/>
+  </ItemGroup>
+
+  <Target Name="_Foo" BeforeTargets="CoreCompile"
```

Comment:
We *really* shouldn't call this target `_Foo`. Please rename to a better name. :-)